QUICK_START.md

```diff
@@ -0,0 +1,160 @@
+```
+
+Respond to the questions with "yes" or "y".
+
+8. Go to http://localhost:8000 and login as admin@example.com.  At
+this point you should be see a list of types of available accelerate
+objects.  Clicking on one of those links should give a you short list
+of some of the objects of the given type.  For example,
+http://localhost:8000/api/impact/Application/
+
+9. Next you need to create a client application.  Click the blue
+"Create An Application" button near the top of the page followed by
+"Register a new application".
+
+10. Give your new client application a name, for "Client type" select
+"Public" and for "Authorization grant type" select "Resource owner
+password-based".  "Redirect URIs" should not be given.  Click "Save".
+
+11. Copy the value under "Client id".  Note that it is a 40 character
+value that may not be completely visible.  However, double clicking it
+and then copying should get the entire value.
+
+12. To functionally test the API, we recommend downloading Postman
+(https://www.getpostman.com/) and importing our testing collection.
+To import a collection into Postman go to File -> Import and select
+the "Import From Link" tab, insert the URL
+https://www.getpostman.com/collections/b9dcecd3190fcd10888d and click
+"Import".
+
+13. Once this collection is loaded, go to the environment ("eye" icon
+near the upper right corner).  Add the global variables "username",
+"password", and "client_id" to the appropriate values from above.
+
+14. Run the "oauth token" API token from the "initialization" folder
+by selecting it and clicking "Send".  This should give a response that
+looks like:
+
+```
+{
+    "access_token": "XXX",
+    "expires_in": 36000,
+    "token_type": "Bearer",
+    "scope": "read write groups",
+    "refresh_token": "XXX"
+}
+```
+
+Postman will automatically parse this an enable the other calls to use
+this access token.
+
+15. To test that you can access the data, open the "low-level" folder,
+select "startup list" and click the blue "Send" button.  This
+automatically selects a startup for subsequent call.  For example, if
+you run the "startup" call, you will see which startup has been
+selected.  Note the "short_pitch".
+
+16. To test that you can change data, select the "startup patch" call.
+Click the "Body" tab on the page.  The "short_pitch" should be
+selected.  If you just click "Send" it will change the "short_pitch"
+returned by the "startup" call.  You can click on the value of
+"short_pitch" in the body of the "startup patch" call and then click
+change to change it to any other value.
+
+17. You can go through a similar process with system users, but
+opening the "v1" folder and using the various "user" calls.  Note that
+there is a "v1 user post" call.  This will create a new user using the
+values given in the body of that call.  Creating a new user will
+automatcally update Postman so it will be the subject of further calls
```

Comment:
Fixed the typo.

More detail seems outside the scope of this ticket which is just to get folks going.  If they want to explore how Postman does what it does, they can do that.